### PR TITLE
Updates Quick Expense article and Swagger with corrected information

### PR DESCRIPTION
**Pull request by jonikamelcher**

### CODE REVIEW

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

File: src/api-explorer/v4-0/QuickExpenses.swagger.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
           "Resources"
         ],
         "summary": "This API is used to create a quick expense. This API does not support image creation.",
-        "description": "This API creates basic expense with minimal fields. Can be added to expense reports from within SAP Concur.",
+        "description": "This API creates basic a expense with minimal fields. Can be added to expense reports from within SAP Concur.",
```
Comment (charlieconcur) [inline src/api-explorer/v4-0/QuickExpenses.swagger.json]: This doesn't quite read correct.

File: src/api-explorer/v4-0/QuickExpenses.swagger.json [hunk 2]
```diff
@@ -113,7 +113,7 @@
           "Resources"
         ],
         "summary": "This API is used to create a quick expense with image. An image is required. This API accepts multipart/form-data request.",
-        "description": "This API creates new quick expense and attaches it with a new image which is provided with multi-part/form-data request.",
+        "description": "This API creates a new quick expense and attaches and image to it, which is provided with multipart/form-data request.",
```
Comment (charlieconcur) [inline src/api-explorer/v4-0/QuickExpenses.swagger.json]: ...attaches an image...

**Review by charlieconcur — CHANGES_REQUESTED** [review]

File: src/api-explorer/v4-0/QuickExpenses.swagger.json (hunk 1 above)
Comment (jonikamelcher) [reply src/api-explorer/v4-0/QuickExpenses.swagger.json]: Corrected.

File: src/api-explorer/v4-0/QuickExpenses.swagger.json (hunk 2 above)
Comment (jonikamelcher) [reply src/api-explorer/v4-0/QuickExpenses.swagger.json]: Corrected.

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Review by charlieconcur — APPROVED** [review]

**Pull request merged**